src/components/fields/textarea/textarea.tsx

```diff
@@ -73,6 +73,8 @@ export const Textarea = (props: IGenericFieldProps<ITextareaSchema>) => {
 				value: newValue,
 			},
 		});
+
+		trigger(id);
```

Comment:
the `onChange` call above should trigger validation already should this is redundant.

this bug appears to happen because user clicked away from the textarea and that triggered the blur event. in the unfortunate instance of the video shared, the effect of the validation happened after the text is appended.

I think a more complete fix will be to prevent the blur event when clicking on any of the pills, you can refer to https://adueck.github.io/blog/keep-focus-when-clicking-on-element-react/ but keep in mind to prevent the blur event ONLY if user clicks the pills and no other elements.